src/schema/schema.py

```diff
@@ -94,6 +110,49 @@ def pretty_repr(self) -> str:
     def pretty_print(self) -> None:
         print(self.pretty_repr())  # noqa: T201
 
+class InterruptMessage(BaseModel):
```

Comment:
Does this actually need to be a whole new class? Why not just make a new `interrupt` type in ChatMessage? It already has a lot of the relevant fields. And could use `custom_data` attribute for any non-general stuff.